packages/suite/src/views/settings/device/FirmwareVersion.tsx

```diff
@@ -69,6 +69,16 @@ export const FirmwareVersion = ({ isDeviceLocked }: FirmwareVersionProps) => {
     const currentFwVersion = getFwVersion(device);
     const availableFwVersion = getFwUpdateVersion(device);
     const { revision } = device.features;
+    const githubUrl = GITHUB_FW_COMMIT_URL + revision;
+    const githubButtonIcon = revision ? 'EXTERNAL_LINK' : undefined;
+
+    const handleAction = () => goto('firmware-index', { params: { cancelable: true } });
```

Comment:
`handleUpdate`?

Reply:
Renamed in 427f67466f36b33deb3c033751dfce922a5e3f00